src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingMechanic.java

```diff
@@ -91,40 +92,45 @@ public Skill getSecondarySkill()
     @Override
     public boolean canRecipeBeAdded(final IToken token)
     {
-        if(!super.canRecipeBeAdded(token))
+        ResourceLocation builder_products = new ResourceLocation("minecolonies", this.getJobName().toLowerCase().concat("_product"));
+        ResourceLocation builder_ingredients = new ResourceLocation("minecolonies", this.getJobName().toLowerCase().concat("_ingredient"));
+
```

Comment:
Those have to be registered somewhere I believe and then be constants there, but the rest looks correct ys.

Reply:
It works with it the way it is unless you are pointing out something I am missing.

Reply:
@OrionDevelopment I'm not 100% sure, wasn't there a registering steps with custom tags too?